res/matrix-org/invite_template.eml.j2

```diff
@@ -0,0 +1,145 @@
+Date: {{ date }}
+From: {{ from }}
+To: {{ to }}
+Message-ID: {{ messageid }}
+Subject: {{ subject_header_value }}
+MIME-Version: 1.0
+Content-Type: multipart/alternative;
+	boundary="{{ multipart_boundary }}"
+
+--{{ multipart_boundary }}
+Content-Type: text/plain; charset=UTF-8
+Content-Disposition: inline
+
+Hi,
+
+{{ sender_display_name }} {{ bracketed_verified_sender }}has invited you into a room
+{{ bracketed_room_name }}on Matrix. To join the conversation, either pick a
+Matrix client from https://matrix.org/docs/projects/try-matrix-now.html or use
+the single-click link below to join via Element requires Chrome, Firefox,
+Safari, iOS or Android
```

Comment:
missing the `(` `)` around 'requires Chrome, ..... Android`, not sure how they got lost